### ISSUE TITLE
Tests against total

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#82]
Currently we can only perform intra-variable tests. We've brainstormed how we might eventually get to inter-variable testing (using an ``array`` mask), but we need to put all of those complications to the side and instead focus on the much more frequently required test-against-total.

Assuming this could be implemented without also trying to solve properly custom test groups between arbitrary variables, this could potentially be done as an enhancement to our existing testing approach, where tests against total could become an additional opt-in, such as:

```python
test_views = qp.ViewMapper(
    template={
        'method': qp.QuantipyViews().coltests,
        'kwargs': {
            'mimic': 'askia',
            'test_total': True,
            'iterators': {
                'metric': ['props', 'means'],
                'level': ['low', 'mid', 'high']
            }
        }
    }
)
```

This obviously presents at least one fundamental problem, in that it implies the generation of *two* views instead of one, since it will be generating results in the ``x``/``@`` link as well as the ``x``/``y`` link.

One potential avenue to explore here would be to always show the results for tests against the total in the ``x``/``y`` link instead of trying to also put them into the ``x``/``@`` link (based on which is significantly higher). 

If necessary, a build could pull the result apart again but, given the increased complexity of trying to implement something more conventional, I don't think it should be attempted at this time (consider, especially, the fact that what appears in the total column is contextually dependent on the structure of a chain, which cannot be known at the time of aggregation).

I'm open to suggestions, but ``@H`` could indicate total is significantly higher and ``@L`` could indicate total is significantly lower. 

Alternatively, ``@`` could indicate total is significantly higher (consistent with existing results) and ``@L`` could be a single exceptional case.

Whatever the notation used to describe it, any significance against total should appear first.

**Comment by alextanski** [comment]
I will follow the 

> I'm open to suggestions, but @H could indicate total is significantly higher and @L could indicate total is significantly lower.

route.

**Comment by alextanski** [comment]
![image](https://cloud.githubusercontent.com/assets/13119448/12915633/d4625bbc-cf2c-11e5-89b3-5d395c311e4f.png)

reflects:
![image](https://cloud.githubusercontent.com/assets/13119448/12915644/e483df66-cf2c-11e5-81e1-7e3dc125b4e6.png)

**Comment by alextanski** [comment]
The above is basically hacked together but completely run-able and generalized. I will now make it nicer code-wise.

**Comment by jamesrkg** [comment]
Resolved by #365.
